daemon/daemon.go

```diff
@@ -266,10 +266,16 @@ func Start(sockFile string, c *cli.Context) error {
 	config := &daemonConfig{
 		Root: root,
 	}
-	exists, err := util.ObjectExists(config)
-	if err != nil {
-		return err
+
+	ignoreCfgFile := c.Bool("ignore-config-file")
```

Comment:
I am not sure how this should work. Because each driver would have it's own configuration as well. In which scenario this would be useful?